Hermes/appscale/hermes/stats/producers/rabbitmq_stats.py

```diff
@@ -0,0 +1,87 @@
+""" Fetches RabbitMQ status. """
+import base64
+import json
+import logging
+import socket
+import time
+
+import attr
+from tornado.httpclient import HTTPClient
+
+from appscale.hermes.stats.converter import include_list_name, Meta
+
+# The port used by the RabbitMQ management plugin.
+API_PORT = 15672
+
+# Credentials used to access the RabbitMQ API.
+USER = 'guest'
+PASS = 'guest'
+
+# The endpoint used for retrieving node stats.
+NODES_API = '/api/nodes'
+
+
+class APICallFailed(Exception):
+  """ Indicates that a RabbitMQ API call failed. """
+  pass
+
+
+@include_list_name('rabbitmq')
+@attr.s(cmp=False, hash=False, slots=True, frozen=True)
+class RabbitMQNodeStats(object):
+  """ The fields reported for each RabbitMQ node. """
+  disk_free_alarm = attr.ib()
+  mem_alarm = attr.ib()
+  name = attr.ib()
+
+
+@attr.s(cmp=False, hash=False, slots=True, frozen=True)
+class RabbitMQStatsSnapshot(object):
+  """ A container for holding RabbitMQ node stats. """
+  utc_timestamp = attr.ib()
+  rabbitmq_node_stats = attr.ib(metadata={Meta.ENTITY_LIST: RabbitMQNodeStats})
```

Comment:
@cdonati , I don't think that nested list is required here. Do we expected that there can be multiple node stats objects? Anyway, `RabbitMQStatsSource.get_current` method puts single object into this attribute.
So I would say that `RabbitMQNodeStats` and `RabbitMQStatsSnapshot` can be merged into single class with node stats and timestamp.